src/logic/resources-status.ts

```diff
@@ -0,0 +1,40 @@
+import { AppComponents } from '../types'
+import os from 'node-os-utils'
+
+export type IResourcesStatusComponent = {
+  areResourcesOverloaded: () => Promise<boolean>
+}
+
+function isAboveNinetyPercent(measure: number) {
+  return measure > 90
+}
+
+const getResourcersUsage = async (): Promise<{ cpuLoad: number; memLoad: number }> => {
```

Comment:
theres is a typo in `resources` but maybe you can get ride of this function, since it's so small